frontend/projects/ui/src/app/services/marketplace.service.ts

```diff
@@ -110,18 +142,19 @@ export class MarketplaceService implements AbstractMarketplaceService {
     private readonly api: ApiService,
     private readonly patch: PatchDB<DataModel>,
     private readonly config: ConfigService,
+    private readonly clientStorageService: ClientStorageService,
   ) {}
 
-  getKnownHosts$(): Observable<StoreIdentity[]> {
-    return this.knownHosts$
+  getKnownHosts$(filtered = false): Observable<StoreIdentity[]> {
+    return filtered ? this.filteredKnownHosts : this.knownHosts$
   }
 
   getSelectedHost$(): Observable<StoreIdentity> {
     return this.selectedHost$
   }
 
-  getMarketplace$(): Observable<Marketplace> {
-    return this.marketplace$
+  getMarketplace$(filtered = false): Observable<Marketplace> {
```

Comment:
Maybe a comment or something would be helpful, so later it is clear what we filter here at a glance?